### ISSUE TITLE
WP media regenerate command just seems to break

### ISSUE THREAD

**Issue by pmgarman** [ymirapp/cli#23]
Added a thumbnail size, which means I need to regenerate thumbnails. I tried running the CLI command, but it failed, and the logs aren't really that enlightening.

```sh
➜  garman git:(ymir) ✗ ymir wp --environment staging
A new version of the Ymir CLI is available: 1.39.1 → 1.42.0

 Please enter the WP-CLI command to run:
 > --url=mindsize.staging.garman.dev media regenerate --only-missing --yes

Running "wp --url=mindsize.staging.garman.dev media regenerate --only-missing --yes" on "staging" environment


  Running the command failed


wp [--environment ENVIRONMENT] [--async] [--] [<wp-command>...]
```

![CleanShot 2023-09-12 at 11 01 37@2x](https://github.com/ymirapp/cli/assets/502720/3043c680-e968-4901-8bc0-cdb15f7f89de)

**Comment by carlalexander** [comment]
Probably a timeout if you reach 60s. That's the default timeout I put for console commands. Might be worth adding another error message for that tho 😅

**Comment by pmgarman** [comment]
Yeah that seems plausible even though I didn't time it. Clearer error messaging would have made me less confused. Is that timeout global or is it something editable inline when running the command?

**Comment by carlalexander** [comment]
It's a [configuration option](https://docs.ymirapp.com/reference/configuration.html#timeout). Need to change it and redeploy. Setting to `false` removes the timeout, but you're still limited by the 15 minute lambda timeout.
